cli.js

```diff
@@ -3,19 +3,21 @@
 'use strict';
 const childProcess = require('child_process');
 const isCi = require('is-ci');
+const isWindows = process.platform === 'win32';
+const npmExec = isWindows ? 'npm.cmd' : 'npm'
```

Comment:
Should we still (try to) use the `npm_execpath` env var? Seems like we'd need it if we wanted to also support yarn or an `npm` that isn't part of PATH. Maybe just use it if not on Windows?